src/code_migrations.rs

```diff
@@ -521,3 +525,25 @@ async fn initialize_local_site_2022_10_10(
 
   Ok(())
 }
+
+async fn merge_site_inbox_2023_11_07(
```

Comment:
I still think it'd be a good idea to move this to SQL, you should be able to do all this below with string manipulation and joining to the instance table to get the site url.

Reply:
Done, but I cant guarantee that the query is correct.